module/witcher.js

```diff
@@ -16,12 +16,14 @@ function updateDerived(actor){
     }
     let thisActor = actor;
     let base = Math.floor((thisActor.data.data.stats.body.current + thisActor.data.data.stats.will.current)/2);
+	let baseMax = Math.floor((thisActor.data.data.stats.body.max + thisActor.data.data.stats.will.max)/2);
```

Comment:
I did calculated the wound treshold automaticly first when I implemented it, but this was problematic with specific values when yoour Hp would go lower than your wound Trheshold, yours stats changed so this  could impact your wound treshold and make it less so your HP becomes above your wound Treshold So it will apply and remove the stats modifiers constatly.  I will test but i'm pretty sure you will have the same problem

Reply:
Thats why I finaly let the woundTreshold has a field that the user has to input

Reply:
I made it so it look at the max body/will instead of the current witch is changed when you get low HP, So it should be not loop.

Reply:
Fixed